apps/web/src/app/admin-console/organizations/policies/personal-ownership.component.html

```diff
@@ -2,15 +2,9 @@
   {{ "personalOwnershipExemption" | i18n }}
 </bit-callout>
 
-<div class="form-group">
-  <div class="form-check">
-    <input
-      class="form-check-input"
-      type="checkbox"
-      id="enabled"
-      [formControl]="enabled"
-      name="Enabled"
-    />
-    <label class="form-check-label" for="enabled">{{ "turnOn" | i18n }}</label>
-  </div>
-</div>
+<ng-container [formGroup]="data">
+  <bit-form-control>
+    <input type="checkbox" bitCheckbox formControlName="enabled" id="enabled" />
```

Comment:
You can bind this to the base component formControl

```suggestion
    <input type="checkbox" bitCheckbox [formControl]="enabled" id="enabled" />
```